qui/updater/utils.py

```diff
@@ -135,10 +135,22 @@ class QubeClass(Enum):
     AppVM = 3
     DispVM = 4
 
+__EffectiveCssProvider__: Gtk.CssProvider = None
+
+def SetEffectiveCssProvider(CssProvider: Gtk.CssProvider) -> None:
```

Comment:
code style comment - it would be better to use all lowercase with undescore.

Reply:
Applied the suggestion